bindings/python/manif/src/Module.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file Module.cpp
+ * @authors Giulio Romualdi, Diego Ferigo
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <pybind11/pybind11.h>
+
+#include <BipedalLocomotion/bindings/manif/Module.h>
+
+#include <BipedalLocomotion/bindings/manif/BaseTypes.h>
+
+namespace BipedalLocomotion
+{
+namespace bindings
+{
+namespace manif
+{
+void CreateModule(pybind11::module& module)
```

Comment:
Sometimes `PYBIND11_EXPORT` is missing, it is necessary?

Reply:
Necessary only in the header file :)